AWSCognitoAuth/AWSCognitoAuth.h

```diff
@@ -105,6 +105,27 @@ typedef void (^AWSCognitoAuthSignOutBlock)(NSError * _Nullable error);
  */
 + (void)removeCognitoAuthForKey:(NSString *)key;
 
+/**
+ Launches the signin UI and updates the session after the user logs in. Current user data is cleared
```

Comment:
This makes it sound like "Current user data is cleared" *after* sign in, which I assume is not correct?